### ISSUE TITLE
Mobetta says someone edited the file when clearly thats not the case

### ISSUE THREAD

**Issue by joeribekker** [maykinmedia/mobetta#10]
Perhaps its really specific, so here's the scenario:

**Entry in PO-file**
```
#, python-format
msgid ""
"\n"
"  %(num_created)s assignments were created\n"
msgstr ""
"\n"
"%(num_created) toewijzingen zijn aangemaakt.\n"
```

Something went wrong (perhaps a new issue) and the translation was missing a "s" character after the right parenthesis. When I tried to add it, via the Mobetta interface, it said someone else edited it before me.

**Comment by JostCrow** [comment]
It could also be the spaces at the beginning of the translation. (the same error was there with enters)

**Comment by sergei-maertens** [comment]
Should be fixed in #20

**Comment by sergei-maertens** [comment]
Fixed in [0.2.9](https://github.com/maykinmedia/mobetta/releases/tag/0.2.9)
